packages/cli/src/ceramic-daemon.ts

```diff
@@ -176,23 +176,17 @@ export class CeramicDaemon {
 
     const ceramic = new Ceramic(modules, params)
     await ceramic._init(true, true)
-
     const did = new DID({
       resolver: {
         ...KeyDidResolver.getResolver(),
         ...ThreeIdResolver.getResolver(ceramic),
         ...NftDidResolver.getResolver({
           ceramic: ceramic,
+          ...opts.didResolvers?.nftDidResolver,
         }),
-        ...(ceramicConfig.ethereumRpcUrl &&
-          EthrDidResolver.getResolver({
-            networks: [
-              {
-                name: 'mainnet',
-                rpcUrl: ceramicConfig.ethereumRpcUrl,
-              },
-            ],
-          })),
```

Comment:
Hm, what happened to the default here now? Are there no default anymore?

Reply:
Got it back.